bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -7,8 +7,20 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
     {{- end }}
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- if or .Values.commonAnnotations .Values.checksumAnnotations }}
```

Comment:
I'd follow the same approach we have in other solutions (for example RabbitMQ) and would not create a new `.Values.checksumAnnotations` parameter. I'd simply add the code and remove this "if" line. What do you think?

Reply:
OK, I'll take a look at RabbitMQ and rework it :)